zcash_client_sqlite/src/lib.rs

```diff
@@ -164,6 +164,26 @@ pub(crate) const DEFAULT_UA_REQUEST: UnifiedAddressRequest =
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Default)]
 pub struct AccountId(u32);
 
+impl AccountId {
+    /// Constructs an `AccountId` from a bare `u32` value. The resulting identifier is not
+    /// guaranteed to correspond to any account stored in the database.
+    #[cfg(feature = "unstable")]
+    pub fn from_u32(value: u32) -> Self {
```

Comment:
Should this be `unsafe`?
```suggestion
    pub unsafe fn from_u32(value: u32) -> Self {
```

Reply:
This isn't unsafe, by any meaning of the term?

Reply:
It's up to the caller to maintain the intended invariant that `AccountId`s are only constructed for accounts stored in the database, which isn't checked.